webknossos/webknossos/dataset/dataset.py

```diff
@@ -717,12 +739,18 @@ def layers(self) -> Dict[str, Layer]:
 
     @property
     def voxel_size(self) -> Tuple[float, float, float]:
-        return self._properties.scale
+        conversion_factor = get_conversion_factor(self._properties.scale.unit)
```

Comment:
self._properties.scale is a VoxelSize, right? Could VoxelSize have a to_nanometer method, returning Tuple[float, float, float]? Then the get_conversion_factor method could be removed (or made private).

Reply:
Yes, I added the method to VoxelSize, looks much better :slightly_smiling_face: